SpringProject/src/main/java/ConferenceRepository.java

```diff
@@ -0,0 +1,4 @@
+package PACKAGE_NAME;
+
+public class ConferenceRepository {
```

Comment:
I assume this will become an interface extending a JpaRepository/CRUDRepository?